### PR TITLE
Start building for Mill 0.11

### PR DESCRIPTION
**Pull request by carlosedp**
This PR removes some unused imports/warnings and starts building for Mill 0.11.

### CODE REVIEW

**Comment by carlosedp** [comment]
Still need to update tests for 0.11.

File: build.sc [hunk 1]
```diff
@@ -13,7 +13,7 @@ import mill.scalalib.api.Util.scalaNativeBinaryVersion
 import mill.scalalib.publish.{Developer, License, PomSettings, VersionControl}
 import scalalib._
 
-val millVersions                           = Seq("0.10.12")
+val millVersions                           = Seq("0.10.12", "0.11.0-M7")
```
Comment (defung) [inline build.sc:16]: should remove the `-M7` suffix
```suggestion
val millVersions                           = Seq("0.10.12", "0.11.0")
```

Comment (carlosedp) [reply build.sc]: Done!

**Comment by joan38** [comment]
Superseded by #144
Thanks @carlosedp and @defung this PR helped me for #144

**Comment by carlosedp** [comment]
Awesome, great to know @joan38

**Pull request closed without merging**